js/map.js

```diff
@@ -29,7 +61,7 @@ map.on("load", () => {
     id: "ihs-fills",
     type: "fill",
     source: "ihs_rollup_source",
-    "source-layer": "ihs_rollup-2jzfoj",
+    "source-layer": "ihs_rollup_categorized-bu6v14",
```

Comment:
I uploaded a new geojson file with each area categorized as being in either the city or the suburbs